dotnet/targets/Xamarin.Shared.Sdk.props

```diff
@@ -105,4 +126,22 @@
 	<PropertyGroup Condition="'$(MtouchUseLlvm)' == '' And '$(Configuration)' == 'Release' And ('$(_PlatformName)' == 'iOS' Or '$(_PlatformName)' == 'tvOS')">
 		<MtouchUseLlvm>true</MtouchUseLlvm>
 	</PropertyGroup>
+
+	<!-- Various options when using NativeAOT -->
+	<PropertyGroup Condition="'$(PublishAot)' == 'true' And '$(_IsPublishing)' == 'true'">
+		<!-- Disable our own assembly IL stripping logic, because ILC does that already -->
+		<EnableAssemblyILStripping>false</EnableAssemblyILStripping>
+
+		<!-- We're using our own native main function when using NativeAOT -->
+		<CustomNativeMain>true</CustomNativeMain>
+
+		<!-- We must find the BCL libraries using the runtime pack instead of using the built-in NativeAOT BCL -->
+		<PublishAotUsingRuntimePack>true</PublishAotUsingRuntimePack>
```

Comment:
Does this mean that to deploy an application with NativeAOT one should only include `-p:PublishAot=true` when publishing:
- via .NET CLI:
    ```bash
    dotnet publish -f net8.0-ios -c Release -r ios-arm64 --self-contained -p:PublishAot=true
    ```
- via .csproj
    ```xml
    <PropertyGroup Condition="'$(Configuration)' == 'Release'">
        <PublishAot>true</PublishAot>
    </PropertyGroup>
    ```
in order to switch between Mono and NativeAOT deployments?

Reply:
There's a plan to use `PublishAot` for Mono as well (https://github.com/dotnet/sdk/issues/25392) so there'd be another property to choose between Mono/NativeAOT

Reply:
No, you can set `PublishAot` always, but it'll only take effect when using `dotnet publish` (to match how `PublishAot` works on other platforms).

Reply:
> No, you can set `PublishAot` always, but it'll only take effect when using `dotnet publish` (to match how `PublishAot` works on other platforms).

Does that mean that if the project file is configured with:
```xml
<PropertyGroup Condition="'$(Configuration)' == 'Release'">
    <PublishAot>true</PublishAot>
</PropertyGroup>
```
doing `dotnet build -c Release` will use MonoAOT, while `dotnet publish -c Release` will use NativeAOT, or I misunderstood this?

Reply:
Correct, and you don't even need the condition:

```xml
<PropertyGroup>
    <PublishAot>true</PublishAot>
</PropertyGroup>
```

This will use MonoAOT:

> dotnet build

While this will use NativeAOT:

> dotnet publish